examples/rbd/fs-pod-rox.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pod-fs-rx-volume
```

Comment:
s/rx/rox .. or iow, thats the accessmode.

Reply:
I wanted to use the template for both ROX and RWX hence I named it RX intentionally.

What IOW stands for ?

Reply:
if the plan is to use the same template for both (rox and rwx) , may be we dont need rx over there.  `r` ->read, `w`->write`, `x` -many is the abbreviation when it comes to accesmode. so `rx` is confusing. so better to remove `rx` from the name itself

Reply:
We already have pod.yaml and raw-block-pod.yaml already for ROX.

If  pod-fs-rx-volume.yaml and pod-block-rx-volume.yaml are confusing should I name them fs-vol-deployment.yaml and block-vol-deployment.yml ?